pkg/api/mlflow/service/artifact/storage/storage.go

```diff
@@ -31,13 +32,15 @@ func (o ArtifactObject) IsDirectory() bool {
 	return o.IsDir
 }
 
-// ArtifactStorageProvider provides and interface to work with particular artifact storage.
+// ArtifactStorageProvider provides an interface to work with artifact storage.
 type ArtifactStorageProvider interface {
+	// Get will return a io.ReadCloser for specific artifact.
+	Get(artifactURI, path string) (io.ReadCloser, error)
```

Comment:
for a future PR: all `ArtifactStorageProvider` methods should take a `context.Context`

Reply:
done in #405